### PR TITLE
Add latest SR1 LZ exclusion limits for WIMP search

### PR DESCRIPTION
**Pull request by ibles**
Changes:
- Add a new file with the SI exclusion limit constrained at the -1 sigma band
- Add new files with the SD exclusion limits that include the bands showed in Fig. 7 and Fig. 8 in https://arxiv.org/abs/2207.03764

### CODE REVIEW

File: data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+6.12000e+02 6.12000e+02 5.82253e-42 1.19181e-41 2.49416e-41 4.62117e-41 7.07591e-41 7.70188e-41 2.04137e-41
+7.16000e+02 7.16000e+02 6.54451e-42 1.37376e-41 2.96752e-41 5.34004e-41 8.17936e-41 8.96552e-41 2.46678e-41
+8.62000e+02 8.62000e+02 8.09836e-42 1.66308e-41 3.53124e-41 6.39404e-41 9.76279e-41 1.05332e-40 2.92159e-41
+1.00800e+03 1.00800e+03 9.36865e-42 1.88000e-41 4.04923e-41 7.48321e-41 1.13538e-40 1.25845e-40 3.41986e-41
+1.21400e+03 1.21400e+03 1.11054e-41 2.31007e-41 4.91454e-41 8.91463e-41 1.36836e-40 1.52425e-40 4.13885e-41
+1.42000e+03 1.42000e+03 1.33865e-41 2.70491e-41 5.83012e-41 1.06694e-40 1.62034e-40 1.75903e-40 4.75082e-41
+1.71000e+03 1.71000e+03 1.63064e-41 3.14984e-41 7.00930e-41 1.25114e-40 1.91091e-40 2.08317e-40 5.79407e-41
+2.00000e+03 2.00000e+03 1.83104e-41 3.73688e-41 8.18927e-41 1.47614e-40 2.26286e-40 2.45817e-40 6.84680e-41
+5.00000e+03 5.00000e+03 4.72335e-41 9.25169e-41 2.02970e-40 3.58810e-40 5.60517e-40 6.15656e-40 1.71706e-40
+1.00000e+04 1.00000e+04 9.70915e-41 1.83774e-40 4.04142e-40 7.23237e-40 1.11227e-39 1.25377e-39 3.42043e-40
```
Comment (kdund) [inline data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt:41]: Why does this file have two mass columns? (or am I mis-reading it?)

Comment (kdund) [reply data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt:41]: The LZ arxiv UL is 4.87e-40 @ 10TeV (the sensi numbers agree though-- was the limit overwritten by the mass?)

**Review by kdund — CHANGES_REQUESTED** [review]
Hi, @ibles thanks! Is it possible that the limit column was overwritten with a mass column?

File: data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt (hunk 1 above)
Comment (kdund) [reply data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt:41]: oh and sorry for getting to this only now :)

File: data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt [hunk 2]
```diff
@@ -0,0 +1,41 @@
+mass limit -2sigma -1sigma median +1sigma +2sigma -nucl.unc. +nucl.unc
+9.00000e+00 2.43625e-41 3.74963e-41 6.22428e-41 1.17659e-40 1.79306e-40 1.42237e-40 4.09002e-41
```
Comment (kdund) [inline data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt:2]: Hi, @ibles I think the limit is still missing here, when comparing to the arxiv version (https://arxiv.org/src/2207.03764v3/anc/Fig7_SpinDependentneutronLimitandSensitivity.txt) there is one column missing 

(limit at 4.8e-41 from that file)

**Review by kdund — CHANGES_REQUESTED** [review]

File: data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt (hunk 2 above)
Comment (ibles) [reply data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt:2]: You are totally right! Sorry, I missed it. Let me fix that.

Comment (ibles) [reply data/LZ_SR1_SpinDependentneutronLimitandSensitivity_corrected.txt]: Good to go. Thanks for catching that Knut!

**Review by kdund — APPROVED** [review]
thanks, @ibles !

**Pull request merged**